File: examples/eApi v1.9/java/eAPI-v19-connector/src/main/java/cz/monetplus/mips/eapi/v19/connector/entity/MallpayAddress.java
package cz.monetplus.mips.eapi.v19.connector.entity;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
@JsonInclude(value = JsonInclude.Include.NON_NULL)
public class MallpayAddress extends ApiBase implements Signable {
    private String name;
    private String country;
    private String city;
    private String streetAddress;
    private String streetNumber;
    private String zip;
    private String addressType;

    @Override
    public String toSign() {
        StringBuilder sb = new StringBuilder();
        add(sb, name);
        add(sb, country);
        add(sb, city);
        add(sb, streetAddress);
        add(sb, streetNumber);
        add(sb, zip);
        add(sb, addressType);
        deleteLast(sb);
        return sb.toString();
    }
}

File: examples/eApi v1.9/dotNET/CsobGTWDemo/CsobGatewayClientExample/Communication/DataObjects/MallpayAddress.cs
using System.Text;
using Newtonsoft.Json;

namespace CsobGatewayClientExample.Communication.DataObjects;

public class MallpayAddress : ApiBase
{
    [JsonProperty("name")] public string? Name { get; set; }

    [JsonProperty("country")] public string? Country { get; set; }

    [JsonProperty("city")] public string? City { get; set; }

    [JsonProperty("streetAddress")] public string? StreetAddress { get; set; }

    [JsonProperty("streetNumber")] public string? StreetNumber { get; set; }

    [JsonProperty("zip")] public string? Zip { get; set; }

    [JsonProperty("addressType")] public string? AddressType { get; set; }


    public string? ToSign()
    {
        var sb = new StringBuilder();
        Add(sb, Name);
        Add(sb, Country);
        Add(sb, City);
        Add(sb, StreetAddress);
        Add(sb, StreetNumber);
        Add(sb, Zip);
        Add(sb, AddressType);
        DeleteLast(sb);
        return sb.ToString();
    }
}